nomenclature/processor/data_validator.py

```diff
@@ -91,30 +110,38 @@ def from_file(cls, file: Union[Path, str]) -> "DataValidator":
 
     def apply(self, df: IamDataFrame) -> IamDataFrame:
         error_list = []
+        warning_list = []
 
-        with adjust_log_level():
+        with adjust_log_level(level="WARNING"):
```

Comment:
The issue is that pyam will print a lot of "filtered data is empty" if you are validating items that don't exist in the data. "WARNING" is the default of `adjust_log_level()`, I think, so this change makes it more explicit.

Reply:
The default is "ERROR". The initial change was made because I was under the impression this log level would reflect on nomenclature. If not the case, it can be reverted.

Reply:
You are right, and you can use this utility to also change other loggers, but the default only touches pyam. 

https://github.com/IAMconsortium/pyam/blob/fed9410aacfa73785c2c01f42564cfaf09b41b76/pyam/logging.py#L21

And to repeat it once more, the reason for silencing the logger here is that pyam writes a warning if you validate but no rows match the criteria - which makes sense from an individual-validation perspective but not in terms of a structured validation-list checked via nomenclature.

https://github.com/IAMconsortium/pyam/blob/fed9410aacfa73785c2c01f42564cfaf09b41b76/pyam/validation.py#L33